.vib/solr/cypress/cypress/integration/solr_test_suite.js

```diff
@@ -0,0 +1,96 @@
+/// <reference types="cypress" />
+import { random, getBasicAuthHeader } from "./utils";
+
+it("allows accessing the Dashboard", () => {
+  cy.login();
```

Comment:
I would add a login failure assertion here due to easier debugging. 

```suggestion
  cy.login();
  cy.get('.login-error').should('not.exist');
```